.circleci/config.yml

```diff
@@ -349,10 +349,10 @@ workflows:
                           # Beta docs are only published from main
                           only: main
             - visual:
-                  name: << matrix.theme >>-<< matrix.color >>-<< matrix.scale >>-<< matrix.dir >>
+                  name: << matrix.system >>-<< matrix.color >>-<< matrix.scale >>-<< matrix.dir >>
                   matrix:
                       parameters:
-                          theme: [spectrum, express]
+                          system: [spectrum, express]
                           color: [lightest, light, dark, darkest]
```

Comment:
Great call - I think this a good opportunity to do it. I am not too concerned with `lightest` and `darkest`, given the current adoption rate and it will reduce the noise for the tests that really matter.